spec.emu

```diff
@@ -3439,16 +4156,17 @@ contributors: Ron Buckton, Ecma International
         </ul>
 
         <emu-clause id="sec-suppressederror">
-          <h1>SuppressedError ( _error_, _suppressed_, _message_ )</h1>
+          <h1>SuppressedError ( _error_, _suppressed_, _message_ [ , _options_ ] )</h1>
           <p>This function performs the following steps when called:</p>
           <emu-alg>
             1. If NewTarget is *undefined*, let _newTarget_ be the active function object; else let _newTarget_ be NewTarget.
             1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%SuppressedError.prototype%"*, « [[ErrorData]] »).
             1. If _message_ is not *undefined*, then
-              1. Let _msg_ be ? ToString(_message_).
-              1. Perform CreateNonEnumerableDataPropertyOrThrow(_O_, *"message"*, _msg_).
-            1. Perform ! CreateNonEnumerableDataPropertyOrThrow(_O_, *"error"*, _error_).
-            1. Perform ! CreateNonEnumerableDataPropertyOrThrow(_O_, *"suppressed"*, _suppressed_).
+              1. Let _messageString_ be ? ToString(_message_).
+              1. Perform CreateNonEnumerableDataPropertyOrThrow(_O_, *"message"*, _messageString_).
+            1. Perform ? InstallErrorCause(_O_, _options_).
+            1. Perform CreateNonEnumerableDataPropertyOrThrow(_O_, *"error"*, _error_).
+            1. Perform CreateNonEnumerableDataPropertyOrThrow(_O_, *"suppressed"*, _suppressed_).
```

Comment:
i think maybe this shouldn't be here?

Reply:
Yeah, copy paste error. The async proposal's spec hadn't yet been amended for the consensus to remove `cause`.